Frontend/ui-customer/src/Screens/Types.purs

```diff
@@ -619,6 +620,10 @@ type DisabilityData = {
 type HomeScreenStateProps =
   {
     currentStage :: Stage
+  , isAutoComplete :: Boolean
```

Comment:
can you move all the props and data regarding searchlocation model to a separate type like searchLocationModelData, searchLocationModelProps.